app/models/child.rb

```diff
@@ -47,6 +48,13 @@ def age(date = Time.current)
     birthday_passed ? years_since_birth : years_since_birth - 1
   end
 
+  def age_in_months(date = Time.current)
```

Comment:
Since NE's rates are based on age in months instead of in years, I added this.  We might want to do months more universally